app/models/sales_log.rb

```diff
@@ -132,6 +132,14 @@ def income1_used_for_mortgage?
     inc1mort == 1
   end
 
+  def buyer_two_will_live_in_property?
+    buy2livein == 1
+  end
```

Comment:
The title of this question is: Were all buyers living at the same address at the time of purchase?

Does answering "Yes" here match this method name?

Reply:
There are two confusingly similarly named sales log fields here called buy2living which is the answer to the question you quote here. and buy2livein which refers to the question "Will buyer 2 live in the property?"

I believe it is strongly recommended against the fields having more readable names, so my instinct is to wrap/hide that obscurely names field inside descriptive methods as soon as possible